openfe_gromacs/protocols/gromacs_md/md_settings.py

```diff
@@ -0,0 +1,668 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe-gromacs
+
+"""Settings class for plain MD Protocols using Gromacs + OpenMMTools
+
+This module implements the settings necessary to run MD simulations using
+:class:`openfe_gromacs.protocols.gromacs_md.md_methods.py`
+
+"""
+from typing import Literal, Optional
+
+from gufe.settings import OpenMMSystemGeneratorFFSettings, SettingsBaseModel
+from openfe.protocols.openmm_utils.omm_settings import (
+    IntegratorSettings,
+    OpenFFPartialChargeSettings,
+    OpenMMEngineSettings,
+    OpenMMSolvationSettings,
+    Settings,
+)
+from openff.models.types import FloatQuantity
+from openff.units import unit
+
+try:
+    from pydantic.v1 import validator
+except ImportError:
+    from pydantic import validator  # type: ignore[assignment]
```

Comment:
```suggestion
from pydantic.v1 import validator
```
This should work now.